src/v3/src/img/smartCardButtonIcon.svg

```diff
@@ -0,0 +1,4 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" fill="none" viewBox="0 0 24 24">
+  <path fill="#1D1D21" fill-rule="evenodd" d="M6 1h13a1.5 1.5 0 0 1 1.5 1.5v19A1.5 1.5 0 0 1 19 23H6a1.5 1.5 0 0 1-1.5-1.5v-19A1.5 1.5 0 0 1 6 1ZM3.5 2.5A2.5 2.5 0 0 1 6 0h13a2.5 2.5 0 0 1 2.5 2.5v19A2.5 2.5 0 0 1 19 24H6a2.5 2.5 0 0 1-2.5-2.5v-19ZM11.502 15h-1v1h1v-1Zm-3 0h1v1h-1v-1Zm0 5v-3h3v1h-2v2h-1Zm2-1h1v1h-1v-1Zm-2-5a1 1 0 0 0-1 1v5a1 1 0 0 0 1 1h3a1 1 0 0 0 1-1v-5a1 1 0 0 0-1-1h-3Z" clip-rule="evenodd"/>
```

Comment:
Should this be `fill="currentColor"` to inherit the color? Just curious

Reply:
good point I think it does so the icon changes color when the user hovers over the button? @alexdahl-okta

Reply:
Sorry I was late here! That's cool, good idea I hadn't thought of. Should be good!